pkg/docker/config/config.go

```diff
@@ -85,6 +86,9 @@ func SetCredentials(sys *types.SystemContext, key, username, password string) (s
 					}
 					return false, setAuthToCredHelper(ch, key, username, password)
 				}
+				if auths.CredsStore != "" {
+					return false, setAuthToCredHelper(auths.CredsStore, key, username, password)
+				}
```

Comment:
Please keep the order of handling `CredHelpers`/`CredsStore`/`AuthConfigs` consistent throughout the package, to make it easier to check that all operations handle the same data. (I guess in the order above, unless there’s a specific reason to do something else.)